sbhs/models.py

```diff
@@ -146,6 +150,7 @@ class Slot(models.Model):
     end_time = models.DateTimeField("End time of a slot",
                 default=timezone.now()+timedelta(
                     minutes=settings.SLOT_DURATION))
+    slot_type = models.CharField(max_length=10, choices=SLOT_TYPES)
```

Comment:
Keep as type instead of slot type, because type will the attribute of the class Slot, so it's understood that type means type of slot.